src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -887,6 +892,7 @@ def flexible_server_connection_string(
 # Custom functions for identity
 def flexible_server_identity_assign(cmd, client, resource_group_name, server_name, identities):
     validate_resource_group(resource_group_name)
+    validate_citus_cluster(cmd, resource_group_name, server_name)
```

Comment:
I do not believe identity commands need to be blocked. These are commands dealing with Microsoft Entra Identity from my understanding which we are able to have enabled from portal. Please verify by trying cli commands on cluster.